pkg/systemd/index.html

```diff
@@ -73,12 +73,18 @@
 
     <div id="server" class="container-fluid page-ct server-overview">
         <div id="motd-box" class="motd-box" hidden>
-          <div class="alert alert-info">
-            <button type="button" class="close" aria-hidden="true">
-              <span class="pficon pficon-close"></span>
-            </button>
-            <span class="pficon pficon-info"></span>
-            <pre id="motd"></pre>
```

Comment:
this was styled rather different. (background, also margin on bottom...)
Before:
![andbefore](https://user-images.githubusercontent.com/12330670/68293051-72861500-008d-11ea-9c59-621b4a1dc2c9.png)

With this PR:
![motdisdifferent](https://user-images.githubusercontent.com/12330670/68292357-fb03b600-008b-11ea-9da6-920332551032.png)

Reply:
Yeah - I am not sure what we should do about the contrast.
It's not only here - modals have the same contrast as well since the background it white.

I would like to get the opinion of @garrett or @andreasn regarding if we should leave this as is or fix it (and not follow the PF4 style)

(I will fix the margin)

Reply:
Contrast is controlled by Patternfly itself, so I would file an issue against that component upstream. Don't feel like it's worth maintaining a local override for this.

Reply:
Margin fixed.

Reply:
I agree about the lack of contrast. I also agree with @andreasn that it should also be fixed upstream.

When there are immediate PF4 issues and we need immediate fixes, we do sometimes override things locally. (But it should still be filed and fixed upstream regardless.)